src/rrs/init/wait.py

```diff
@@ -169,16 +188,27 @@ def rr_enabled_and_setup() -> bool:
         return False
 
     logger.info("Checking zoning for Kubernetes and CEPH nodes...")
+    if kubernetes_zones_exist():
+        logger.info("Kubernetes zones are created.")
+    else:
+        logger.info("Not deploying the cray-rrs chart.")
+        return False
+
     if ceph_zones_exist():
         logger.info("CEPH zones are created.")
     else:
         logger.info("CEPH zones are not created.")
         return False
 
-    if kubernetes_zones_exist():
-        logger.info("Kubernetes zones are created.")
+    try:
+        rollout_status = restart_completed()
```

Comment:
Why do you have this try block here as well as one giant try block inside the restart_completed function? It seems like you'd only need one or the other, right?